BridgeCareApp/BridgeCareCodeFirst/Data/GetTargetCellsData.cs

```diff
@@ -0,0 +1,98 @@
+﻿using BridgeCare.Models;
```

Comment:
Per our discussion, investigate implementation of a ExcelCellShader strategy that will "ShadeCells". See if you can inject the behavior required for actually coloring cells a certain color.